src/containers/FilmFrontpage/MovieGrid.tsx

```diff
@@ -9,48 +9,116 @@
 import { CSSProperties } from "react";
 import { useTranslation } from "react-i18next";
 import { gql } from "@apollo/client";
-import styled from "@emotion/styled";
-import { breakpoints, mq, spacing } from "@ndla/core";
-import { Heading } from "@ndla/typography";
+import { Heading, Skeleton } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import FilmContentCard from "./FilmContentCard";
 import { GQLResourceTypeMoviesQuery, GQLResourceTypeMoviesQueryVariables } from "../../graphqlTypes";
 import { useGraphQuery } from "../../util/runQueries";
 
-const StyledSection = styled.section`
-  margin-left: ${spacing.normal};
-  margin-right: ${spacing.normal};
-  ${mq.range({ from: breakpoints.desktop })} {
-    margin-left: ${spacing.xlarge};
-    margin-right: ${spacing.xlarge};
-  }
-`;
+const StyledSection = styled("section", {
+  base: {
+    display: "flex",
+    justifyContent: "center",
+  },
+});
 
-const MovieListing = styled.div`
-  display: flex;
-  flex-wrap: wrap;
-  gap: ${spacing.normal};
-  margin: ${spacing.small} 0;
-`;
+const ColumnWrapper = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+  },
+});
 
-const StyledFilmContentCard = styled(FilmContentCard)`
-  opacity: 0;
-  animation-fill-mode: forwards;
-  animation-name: fadeIn;
-  animation-duration: 300ms;
-  animation-delay: calc(var(--index) * 50ms);
-`;
+const MovieListing = styled("div", {
+  base: {
+    display: "grid",
+    gap: "medium",
+    gridTemplateColumns: "repeat(4, 1fr)",
+    desktopDown: { gridTemplateColumns: "repeat(3, 1fr)" },
+    tabletDown: { gridTemplateColumns: "repeat(2, 1fr)" },
+  },
+});
 
-const StyledHeading = styled(Heading)`
-  display: flex;
-  gap: ${spacing.small};
-  margin: ${spacing.xsmall} 0;
-`;
+const StyledFilmContentCard = styled(FilmContentCard, {
+  base: {
+    animationFillMode: "backwards",
+    animationName: "fade-in",
+    animationDuration: "300ms",
+    animationDelay: "calc(var(--index) * 50ms)",
+  },
+});
+
+const StyledHeading = styled(Heading, {
+  base: {
+    display: "flex",
+    gap: "small",
+    paddingBlockEnd: "xsmall",
+  },
+});
+
+const LoadingShimmer = () => {
+  return new Array(24).fill(0).map((_, index) => (
```

Comment:
Kan vi redusere denne litt? 8 eller 12 burde holde.

Reply:
Vi kan, men jeg syns det er penest med minst 20.
Er det ytelsen du er redd for?

Reply:
Ikke nødvendigvis redd, men tenker at brorparten av disse vil legge seg under folden. De fleste vil vel sitte stille og vente på at siden laster?

Reply:
Nja, men det skjer jo når man bytter kategori. Og det vil man jo gjerne gjøre når man har scrollet litt ser jeg for meg.